holidays/countries/nepal.py

```diff
@@ -0,0 +1,263 @@
+
+        # Women's Day.
+        self._add_womens_day("Women's Day")
+
+        # Nepal New Year (Vikram Sambat).
+        self._add_vaisakhi("Nepali New Year (Vikram Sambat)")
+
+        # Labor Day.
+        self._add_labor_day("Labor Day")
+
+        # Republic Day.
+        self._add_holiday_may_29("Republic Day")
+
+        # Constitution Day.
+        self._add_holiday_sep_19("Constitution Day")
+
+        # Christmas.
+        self._add_christmas_day("Christmas")
+
+        # Tamu Losar.
+        self._add_holiday_dec_30("Tamu Losar")
+
+        # Hindu holidays.
+
+        # Maghe Sankranti.
+        self._add_makar_sankranti("Maghe Sankranti")
+
+        # Sonam Losar.
+        self._add_sonam_losar("Sonam Losar")
+
+        # Gyalpo Losar.
+        self._add_gyalpo_losar("Gyalpo Losar")
+
+        # Maha Shivaratri.
+        self._add_maha_shivaratri("Maha Shivaratri")
+
+        # Holi (Mountain & Hilly).
+        self._add_nepal_holi("Holi (Mountain & Hilly)")
+
+        # Holi (Terai).
+        self._add_holi("Holi (Terai)")
+
+        # Ram Navami.
+        self._add_ram_navami("Ram Navami")
+
+        # Buddha Jayanti.
+        self._add_buddha_purnima("Buddha Jayanti")
+
+        # Ghatasthapana.
+        self._add_sharad_navratri("Ghatasthapana")
+
+        # Phulpati.
+        self._add_maha_saptami("Phulpati")
+
+        # Maha Ashtami.
+        self._add_maha_ashtami("Maha Ashtami")
+
+        # Maha Navami.
+        self._add_maha_navami("Maha Navami")
+
+        # Vijayadashami.
+        self._add_dussehra("Vijayadashami")
+
+        # Ekadashi(Dashain).
+        self._add_papankusha_ekadashi("Ekadashi (Dashain)")
+
+        # Duwadashi(Dashain).
+        self._add_papankusha_duwadashi("Duwadashi (Dashain)")
+
+        # Janai Purnima.
+        self._add_raksha_bandhan("Janai Purnima")
+
+        # Shree Krishna Janmashtami.
+        self._add_janmashtami("Shree Krishna Janmashtami")
+
+        # Lakshmi Puja.
+        self._add_diwali_india("Lakshmi Puja")
+
+        # Gai Tihar.
+        self._add_gau_krida("Gai Tihar")
+
+        # Bhai Tika.
+        self._add_bhai_dooj("Bhai Tika")
+
+        # Govardhan Puja
+        self._add_govardhan_puja("Govardhan Puja")
+
+        # Mha Puja.
+        self._add_govardhan_puja("Mha Puja")
+
+        # Tihar holiday.
+        self._add_tihar_holiday("Tihar Holiday")
+
+        # Chhath Parwa.
+        self._add_chhath_puja("Chhath Parwa")
+
+        # Islamic holidays.
+
+        # Eid al-Fitr.
+        self._add_eid_al_fitr_day("Id-ul-Fitr")
+
+        # Eid al-Adha.
+        self._add_eid_al_adha_day("Bakrid")
+
+
+class NepalIslamicHolidays(_CustomIslamicHolidays):
+    # Bakrid / Eid-al-Adha.
+    EID_AL_ADHA_DATES = {
+        2001: (MAR, 6),
+        2002: (FEB, 23),
+        2003: (FEB, 12),
+        2004: (FEB, 2),
+        2005: (JAN, 21),
+        2006: ((JAN, 11), (DEC, 31)),
+        2007: (DEC, 20),
+        2008: (DEC, 9),
+        2009: (NOV, 28),
```

Comment:
Even if we move the `start_year` to 2010, keeping 2001-2009 record for Islamic holidays as-is wouldn't hurt.

Reply:
If they are just taken from TimeAndDate, then I'm not sure about their usefulness, to be honest.